src/mmds/src/data_store.rs

```diff
@@ -190,7 +150,11 @@ mod tests {
                     "UK": "+44 1234567"
                 },
                 "mobile": "+44 2345678"
-            }
+            },
+            "phones2": [
```

Comment:
This is a bit funky. Why not replace phones or add something else that makes sense?